easysnmp/interface.c

```diff
@@ -1393,7 +1422,7 @@ static long long py_netsnmp_attr_long(PyObject *obj, char *attr_name)
     return val;
 }
 
-static void *py_netsnmp_attr_void_ptr(PyObject *obj, char *attr_name)
+/*static void *py_netsnmp_attr_void_ptr(PyObject *obj, char *attr_name)
```

Comment:
It's better to use `#if 0` -> `#endif` for code that's going to be disabled over the short-term. It's better to just remove it, however, in a separate commit if it's unused (it can be restored later as need be from git).

Using multi-line comments is bad because (depending on the compiler), it can flag the issue and fail builds with `-Werror` enabled.

Reply:
done, thanks